clients/privacy-center/middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from "next/server";
+
+export default function middleware(request) {
+  const start = Date.now();
+  const response = NextResponse.next();
+  const stop = Date.now();
+  const handler_time = stop - start;
+
+  const log_dict = {
+    method: request.method,
+    status_code: response.status,
+    handler_time: `${handler_time}ms`,
```

Comment:
This often comes back as "0" or "1"...is it possible the server is actually that fast with requests that it can deliver in under a millisecond?

Reply:
It looks like that method may not actually process the request and return the response. The docs say `next()`: `Returns a NextResponse that will continue the middleware chain`. I guess it's just a simple function call that returns immediately. 


There is a github discussion about how it's not possible to get the request execution time in a middleware function: https://github.com/vercel/next.js/discussions/34420


That's disappointing. Nextjs may not support it.

Reply:
oh.....well that's fine lol as long as it matches the format we'll make @RobertKeyser's life easier 😄